integrations.json

```diff
@@ -5172,40 +5172,13 @@
           "maximum_patch": 65535
         },
         "wrapper": {
-          "assembly": "Datadog.Trace, Version=1.28.2.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
+          "assembly": "Datadog.Trace, Version=1.28.3.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
           "type": "Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.XUnit.XUnitTestRunnerRunAsyncIntegration",
           "action": "CallTargetModification"
         }
       }
     ]
   },
-  {
```

Comment:
This integration got removed. Is this expected @anna-git @robertpi ?

Reply:
Confirmed, this was expected and was already an open PR. We're closing #1685 and keeping the change in this PR